mitiq/zne/scaling/folding.py

```diff
@@ -289,6 +367,45 @@ def _get_num_to_fold(scale_factor: float, ngates: int) -> int:
 
 
 # Local folding functions
+@noise_scaling_converter
+def fold_all(
+    circuit: QPROGRAM,
+    num_folds: int = 1,
+    exclude: FrozenSet[Any] = frozenset(),
```

Comment:
```suggestion
    exclude: FrozenSet[str] = frozenset(),
```

Reply:
I would keep this `Any` because Cirq gates are also allowed.